src/main/java/com/minecraftabnormals/upgrade_aquatic/core/mixin/DrownedRendererMixin.java

```diff
@@ -0,0 +1,44 @@
+package com.minecraftabnormals.upgrade_aquatic.core.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+import com.mojang.blaze3d.matrix.MatrixStack;
+
+import net.minecraft.client.renderer.entity.AbstractZombieRenderer;
+import net.minecraft.client.renderer.entity.DrownedRenderer;
+import net.minecraft.client.renderer.entity.EntityRendererManager;
+import net.minecraft.client.renderer.entity.model.DrownedModel;
+import net.minecraft.entity.monster.DrownedEntity;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.vector.Vector3f;
+
+@Mixin(DrownedRenderer.class)
+public class DrownedRendererMixin extends AbstractZombieRenderer<DrownedEntity, DrownedModel<DrownedEntity>> 
+{
+	protected DrownedRendererMixin(EntityRendererManager p_i50974_1_, DrownedModel<DrownedEntity> p_i50974_2_, DrownedModel<DrownedEntity> p_i50974_3_, DrownedModel<DrownedEntity> p_i50974_4_) {
+		super(p_i50974_1_, p_i50974_2_, p_i50974_3_, p_i50974_4_);
+	}
+	
+	/**
+	 * @reason when i tried to use inject instead of overwrite it made the entity walk backwards. hence why this is an overwrite
+	 * @author tallest red
+	 */
+	@Overwrite
+    protected void applyRotations(DrownedEntity entityLiving, MatrixStack matrixStackIn, float ageInTicks, float rotationYaw, float partialTicks) 
```

Comment:
If you're going to add comments, have them be grammatically correct. Capitals for starts of sentences and end sentences with periods.

I wouldn't prefer adding `@author` here on this method for only this class. The `@author` should be at the top of the class since the entire class is made by you, but this is still inconsistent since every other class doesn't have an author tag from you.